src/views/dashboard/playground/docs/host-metrics-promql.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Monitor Host Metrics with PromQL
+---
+
+# Monitor Host Metrics with PromQL
+
+GreptimeDB has implemented a set of [Prometheus-compatible APIs](https://docs.greptime.com/user-guide/query-data/promql#prometheus-http-api) that use the same input and output format as the original Prometheus HTTP API. In addition, GreptimeDB also exposes [a custom HTTP API](https://docs.greptime.com/user-guide/query-data/promql#greptimedb-s-http-api) for querying with PromQL and returning GreptimeDB's data frame output.
+
+In this documentation, we will use Prometheus [node_exporter](https://github.com/prometheus/node_exporter) to export machine metrics to GreptimeCloud and query the metrics using PromQL.
+
+## Sample Data
+
+Node_exporter is a Prometheus exporter for hardware and OS metrics exposed by *NIX kernels.
+It is written in Go and has pluggable metric collectors.
+For the metrics it collects, please refer to the [node_exporter Github repository](https://github.com/prometheus/node_exporter).
+
+## Export Data to GreptimeCloud
+
+Spin up a Docker container to write sample data to your database:
+
+```shell
+docker run --rm -e GREPTIME_URL='https://<host>/v1/prometheus/write?db=<dbname>' \
+    -e GREPTIME_USERNAME='<username>' \
+    -e GREPTIME_PASSWORD='<password>' \
+    --name greptime-node-exporter greptime/node-exporter
+```
```

Comment:
I doubt if we can expect user to setup containers when using playground. It breaks our principle of pure browser experience.

Reply:
It would be possible if we can either:

1. import preset data
2. generate fake data
3. use live data from our show-case service

Reply:
Fresh data needs to be written here, so option 2 or 3 are the choices. But how to generate fake data using PromQL in playground? 
Maybe option 3 is a more practical approach, which needs the show-case service to be accessed in playground, how to achieve this?